.github/workflows/check.yaml

```diff
@@ -0,0 +1,33 @@
+name: Check workflow running linter, unit and functional tests
```

Comment:
We need to change `sonar.yaml` ([L#9](https://github.com/canonical/prometheus-juju-backup-all-exporter/blob/main/.github/workflows/sonar.yaml#L9C11-L9C11) and [L#17](https://github.com/canonical/prometheus-juju-backup-all-exporter/blob/main/.github/workflows/sonar.yaml#L17)) to use this workflow name

Reply:
Och I did no realized that and there are potentially more name like this. @dashmage Can I ask you to check to fix it also in other projects (I think [this](https://github.com/search?q=org%3Acanonical%20%22name%3A%20Check%20workflow%20running%20linter%2C%20unit%20and%20functional%20tests%22&type=code) is full lists), thanks.